spring-cloud-gcp-examples/spring-cloud-gcp-trace-example/README.adoc

```diff
@@ -0,0 +1,25 @@
+= Spring Cloud GCP Trace Starter Example
+
+This sample application demonstrates using the
+link:../../spring-cloud-gcp-starters/spring-cloud-gcp-starter-trace[Spring Cloud GCP Trace Starter] in your code.
+
+== Setup & Configuration
+1. Create a Google Cloud Platform Project
+1. https://cloud.google.com/docs/authentication/getting-started#creating_the_service_account[Create a service account]
+   with Trace Append permission. Furnish a new JSON key and then set the credentials using the
+   `GOOGLE_APPLICATION_CREDENTIALS` environment variable or
```

Comment:
Not sure if it's worth mentioning the envvar way.

Reply:
could i trouble you to clarify? GOOGLE_APPLICATION_CREDENTIALS is the env var way?  Or, do you mean env var to override spring.cloud.gcp.credentials-location properties?  (I'd avoid that one since it's better known feature of spring)

Reply:
Yeah, I meant the GOOGLE_APPLICATION_CREDENTIALS way.
If you think it's better to leave it, feel free to leave it.